pyproject.toml

```diff
@@ -1,10 +1,11 @@
 [build-system]
-requires = ["maturin"]
+requires = ["maturin>=0.11,<0.12"]
 build-backend = "maturin"
 
 [tool.maturin]
 bindings = "pyo3"
 manylinux = "2010"
+cargo-extra-args = "--features extension-module,openssl"
```

Comment:
this suggests it *is* the default. do you still need to specify it?

Reply:
Python `pep517` builds look at this file, but `cargo` doesn't. I don't think `maturin` does either.